### DISCUSSION TITLE
Assigning fields in the log file  data/var/log/http-request.log

### DISCUSSION THREAD

**Discussion by qemu-glusterf** [trinodb/trino#24456] [Q&A]
Good afternoon! 
I'm using Trino 457.
Is it possible to decrypt the fields(names) in the file logs. 
         data/var/log/http-request.log
This is necessary for parsing logs in vector

![Image](https://github.com/user-attachments/assets/1261c54b-7a5e-4b94-9f6f-322aebd7183f)

Thanks in advance for the answer!
PS
The most interesting are the names of these fields:
5), 6), 8), 9), 10), 11), 13), 14), 15), 16)

**Comment by sajjoseph** [comment]
You will find details in here:

https://github.com/airlift/airlift/blob/cb8a9ba07ae4d6bd53b12a4b4f74cbd2902d1ffd/http-server/src/main/java/io/airlift/http/server/JettyRequestLog.java#L110

**Comment by qemu-glusterf** [comment]
Thank you very much!
Have a nice day!
